packages/core/typings/js-joda.d.ts

```diff
@@ -2198,7 +2198,8 @@ export class DateTimeFormatterBuilder {
     appendOffset(pattern: string, noOffsetText: string): DateTimeFormatterBuilder;
     appendOffsetId(): DateTimeFormatterBuilder;
     appendPattern(pattern: string): DateTimeFormatterBuilder;
-    appendValue(field: TemporalField, width?: number, maxWidth?: number, signStyle?: SignStyle): DateTimeFormatterBuilder;
+    appendValue(field: TemporalField, width?: number): DateTimeFormatterBuilder;
```

Comment:
I haven't done that yet, because the second argument for the first option is marked as optional, so it's technically not required. But I think it's better "style", so let's add it.

Reply:
Done :+1: